source/includes/mongodb-compatibility-table-php.rst

```diff
@@ -14,163 +15,159 @@
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - 
    * - ext 1.8 + lib 1.7
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - 
    * - ext 1.7 + lib 1.6
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - 
    * - ext 1.6 + lib 1.5
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - 
    * - ext 1.5 + lib 1.4
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - 
    * - ext 1.4 + lib 1.3
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - ext 1.3 + lib 1.2
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - ext 1.2 + lib 1.1
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - ext 1.1 + lib 1.0
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
    * - ext 1.0
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-
-.. [#PHPC1.10-PHPLIB1.9-driver-support] The extension 1.10 + library 1.9
-   driver does not support snapshot reads on secondaries. For more
-   information, see the `MongoDB Server version 5.0 release notes <https://docs.mongodb.com/v5.0/release-notes/5.0/#snapshots>`__. 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓.. [#PHPC1.10-PHPLIB1.9-driver-support] The extension 1.10 + library 1.9   driver does not support snapshot reads on secondaries. For more   information, see the `MongoDB Server version 5.0 release notes <https://docs.mongodb.com/v5.0/release-notes/5.0/#snapshots>`__.
```

Comment:
```suggestion
     - ✓
     
   .. [#PHPC1.10-PHPLIB1.9-driver-support] The extension 1.10 + library 1.9   driver does not support snapshot reads on secondaries. For more   information, see the `MongoDB Server version 5.0 release notes <https://docs.mongodb.com/v5.0/release-notes/5.0/#snapshots>`__.
```

Reply:
Sorry I missed that, thanks for catching.